Tests/NimbleTests/Matchers/SatisfyAllOfTest.swift

```diff
@@ -52,6 +52,7 @@ final class SatisfyAllOfTest: XCTestCase {
         expect(true).toNot(beTruthy() && beFalsy())
     }
 
+    #if !os(WASI)
```

Comment:
Why does this test (and the equivalent in `SatisfyAnyOfTest`) need to be disabled in wasm?

Reply:
That's because `toEventually` relies on Dispatch, which is not available on SwiftWasm.